<!DOCTYPE HTML>
<!--
	Author: @mrprotein24 x chowdhurylab
-->
<html>
	<head>
		<title>Research â€” ChowdhuryLab</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" href="images/rz.jpg">
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<div id="main">
						<div class="inner">

							<!-- Header -->
								<header id="header">
									<a href="index.html" class="logo"><b style="color:#ff8300">ChowdhuryLab</b></a>
									<ul class="icons">
								</header>

							<!-- Content -->
								<section>
									<header class="main">
										<h1 style="color:#7e7e7e; font-family: Helvetica Neue, Helvetica, Arial, sans-serif">Research</h1>
										<p style="color:#939393;font-family: Helvetica Neue, Helvetica, Arial, sans-serif"> What we do</p></br>
									</header>

									<p>Natural proteins are linear, unidirectional biopolymers constructed by strategically adding one of the twenty possible amino acids after another such that they fold into a precise, non-clashing 3D-structure. Proteins come in various lengths, compositions, geometries, and functions. The structure of a protein offers grooves/ patches to fit/ attract small molecules (drugs, ions, etc.) or other proteins. The amino acid sequence dictates the structure and consequently the function that a protein can execute. While experiments can help us discern what the sequence, structure, and function of any protein are, predicting causal sequence-structure-function maps have largely remained elusive. We use a combination of deep-learning networks, physics-based molecular-mechanical models, and integer-optimization to predict amino acid transitions (mutations, insertions, or deletions) that will endow a protein with a desired functionality (with/ without significant structural change). We also predict protein activity given the pH, dielectric constant, polarity, and temperature of the surrounding solvent.</p> </br > 
									<p>We modify known natural/ synthetic proteins or design from scratch proteins for - (a) enhanced/ altered <b>catalysis</b>, (b) precise <b>molecular sieving</b> of ions or solutes, (c) <b>DNA/RNA</b>-sequencing, (d) bio-based <b>fuel cells</b>, (e) <b>biomedical</b> applications and drug discovery, (f) structure-guided <b>metabolic modeling</b>, and (g) extracting specific <b>rare-earth elements</b> from electronic waste.</p></br> 
									<p>We are a <i>computational group</i> but we collaborate with experimental collegues for each of our projects. We also utilize tools from other established domains of quantitative science for analogous modeling of biological systems. </p></br></br>


									<div class="posts">
										<article>
											<h2 style="color:#7e7e7e" align="left">Enzyme design (biocatalysis)</h2>
											<a class="image"><img src="images/enz1.gif" alt="" /></a>
											<p align="left" >We use latent space encoding of both enzymatic protein sequences and their geometry to predict mutations that  functionalize them for intended (non-)reactive chemistry. We identify/ build stable, promiscuous proteins with low-level activity towards our intended biochemical conversion and subsequently predict mutations to crank-up a desired activity.<br/><b style="color:#5680e9">Applications</b> : biosensor design, production of precursors for biofuel and drugs</p>
										</article>

										<article>
											<h2 style="color:#7e7e7e" align="left">Transporter design (separations)</h2>
											<a class="image"><img src="images/enz3.gif" alt="" /></a>
											<p>We use pore-containing proteins as chassis to design precise molecular separation devices. Mutations on the exterior of the pore can be used to tune packing density/ efficiency of these pores in biomimetic membranes while inner pore walls are functionalized to endow them with different solute/ ion selectivities. <br/><b style="color:#5680e9">Applications</b> : solute separation, bio-based fuel cell, genome sequencing</p>
										</article>

										<article>
											<h2 style="color:#7e7e7e" align="left">Antibody design (immunology)</h2>
											<a class="image"><img src="images/enz4.gif" alt="" /></a>
											<p align="left">We design libraries of humanized proteins with complementary shape and electrostatics to neutralize disease-causing antigenic proteins. We profile our predictions based on scoring metrics that account for dynamics of binding, binding affinity, and possibility of antigenic protein to mutate and escape based on evolutionary profiling of antigenic proteins. <br/><b style="color:#5680e9">Applications</b> : antibody design, CDR3 loop engineering</p>
										</article>

										<article>
											<h2 style="color:#7e7e7e" align="left">Strain design (metabolic modeling)</h2>
											<a class="image"><img src="images/enz2.gif" alt="" /></a>
											<p>We predict structures of entire proteome that participate in metabolic pathways. We build and estimate substrate binding and product leaving kinetic parameters using protein-small molecule interaction calculations. Structure-aware metabolic models provide biophysical insights about whole-cell behavior. <br/><b style="color:#5680e9">Applications</b> : microbial engineering, biocontainment of abiotic amino acids</p>
										</article>

									</div>
								</section>

						</div>
					</div>

				<!-- Sidebar -->
					<div id="sidebar">
						<div class="inner">

							<!-- Menu -->
								<nav id="menu">
									<header class="major">
									</header>
									<ul>
										<li><a href="index.html">Home</a></li>
										<li><a href="members.html">Members</a></li>
										<li><a href="research.html">Research</a></li>
										<li><a href="publications.html">Publications</a></li>
										<li><a href="resources.html">Resources</a></li>
										<li><a href="news.html">News</a></li>
										<li><a href="joinus.html">Join Us</a></li>
									</ul>
								</nav>

								<section>
									<header class="major">
											<h2 style="color:#818589">Latest in Lab</h2>
									</header>
									<div class = "mini-posts" style="height:10em;width:20em;overflow:auto;background-color:#dcdcdc;color:black;scrollbar-base-color:gray;font-family:Helvetica Neue;">
										<b style="color:#5680e9;font-family: Helvetica Neue, Helvetica, Arial, sans-serif">Jun '22</b>	<span style="color:#7e7e7e;font-family: Helvetica Neue, Helvetica, Arial, sans-serif">Nick Clark (postdoc at Harvard Medical School) joins the lab. Welcome Nick!</span> <span style="color:#818589" class="icon fa solid fa-user-plus"></span><br \><br />
										<b style="color:#5680e9;font-family: Helvetica Neue, Helvetica, Arial, sans-serif">Apr '22</b>	<span style="color: #7e7e7e;font-family: Helvetica Neue, Helvetica, Arial, sans-serif">Iowa State will be the first home of the Chowdhury Lab from Fall 2022!</span> <span style="color:#818589" class="icon solid fa-star"></span><br \><br />
									</div>
								</section>
							<!-- Section Tidings-->
								<section>
									<header class="major">
										<h2 style="color:#818589">Good Tidings</h2>
									</header>
									<div class="mini-posts">
										<article>
											<a class="image"><img src="images/office.jpeg" alt="" /></a>
											<p>To new beginnings - Prof. Chowdhury will start at CBE @Iowa State from Fall 2022</p>
										</article>
									</div>
								</section>


						</div>
					</div>
			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
